crates/bevy_core_pipeline/src/bloom/settings.rs

```diff
@@ -118,6 +126,8 @@ impl BloomSettings {
             threshold_softness: 0.0,
         },
         composite_mode: BloomCompositeMode::EnergyConserving,
+        max_mip_dimension: 512,
```

Comment:
I'd keep the const around for these instead of hardcoding it. Maybe rename it to `DEFAULT_MAX_MIP_DIMENSION`